recipe/meta.yaml

```diff
@@ -129,21 +128,21 @@ outputs:
       binary_relocation: False
     run_exports:
       strong:
-        - cuda-version >={{ cuda_version }},<{{ cuda_version.split(".")[0]|int + 1 }}
+        - cuda-version >={{ cuda_version }},<{{ cuda_version_next_major }}
     files:                             # [linux]
       - lib/libnvptxcompiler_static.a  # [linux]
-      - nvvm/include                   # [linux]
-      - nvvm/lib64                     # [linux]
     requirements:
       build:
         - sysroot_{{ target_platform }} 2.17    # [linux]
       host:
         - arm-variant * {{ arm_variant_type }}  # [aarch64]
         - cuda-version {{ cuda_version }}
         - cuda-cudart-dev
+        - {{ pin_subpackage("cuda-nvvm-impl", exact=True) }}
```

Comment:
One more possibly unnecessary host (just marking to make them easier to find in case they can be removed).

Reply:
As we are only packaging one static library here that shouldn't be packaged anywhere else, this is probably safe to drop.